src/lib/index.js

```diff
@@ -335,6 +335,23 @@ lib.noneOrAll = function(containerIn, containerOut, attrList) {
     }
 };
 
+/**
+ * Fill with unique items
+ *
+ * @param {array} array
+ *  array to be filled
+ * @param {any} item
+ *  item to be or not to be inserted
+ * @return {array}
+ *  ref to array (now possibly containing one more item)
+ *
+ */
+lib.fillUnique = function(array, item) {
```

Comment:
Maybe inconsequential, but what do you think of `lib.addUnique` instead? 

In my head, `fillUnique` implies _filling_ an entire array of some length.